cli.js

```diff
@@ -1,22 +1,14 @@
 #!/usr/bin/env node
-
 const open = require('open')
 const readline = require('readline')
-const dedent = require('dedent')
 const getRepo = require('./lib/get-repo')
+const handleNoArgs = require('./lib/handle-no-args')
 
 if (process.stdin.isTTY) {
   // Read from arguments
-  const names = process.argv.slice(2)
+  let names = process.argv.slice(2)
   if (!names.length) {
-    console.log(dedent`To use ghub, specify one or many package names.
-      Examples:
-
-      ghub express
-      ghub choo chai chalk`)
-
-    // exit gracefully to avoid dumping a bunch of npm info
-    process.exit()
+    names = handleNoArgs() || process.exit()
```

Comment:
I feel like the `cli.js` was getting too big and thus splitting it into more modules.
Also, I think a critical function such as `process.exit` shouldn't be within another one, and thus remaining here.

Reply:
I mean, the `cli` should be the one deciding whether it exits or not.